apps/emqx_exhook/src/emqx_exhook.app.src

```diff
@@ -1,6 +1,6 @@
 {application, emqx_exhook,
  [{description, "EMQ X Extension for Hook"},
-  {vsn, "4.3.0"},
+  {vsn, "4.3.3"},
```

Comment:
Did you modify the wrong file? 
apps/emqx_management/src/emqx_management.app.src

apps/emqx_management/src/emqx_management.appup.src also needs to be updated

Reply:
Probably this is an issue with the script, it told me to modify this file.

Reply:
there are protobuf files changed in this app's priv dir.
still pending on @HJianBo to check how to proceed

Reply:
fixed here: https://github.com/emqx/emqx/pull/4885
@k32 rebase ?